src/main/java/Duke/Commands/DeleteCommand.java

```diff
@@ -0,0 +1,69 @@
+package Duke.Commands;
+
+import Duke.Errors.DeleteException;
+import Duke.Errors.DukeException;
+import Duke.Errors.FileAbsentException;
+import Duke.Helpers.Storage;
+import Duke.Helpers.TaskList;
+import Duke.Helpers.Ui;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * handles the case when the keyword is delete
+ */
+public class DeleteCommand extends Command {
+    /**
+     * assigns string to a value of string
+     * @param string assigns string to this this.string
+     */
+    public DeleteCommand(String string) {
+        super(string);
+    }
+
+    private String rewrite(Storage storage, TaskList tasks, int ID)  throws DukeException{
+        System.out.println("   Noted. I've removed this task:\n" +
+                "   " + tasks.getAllTasks().get(ID - 1).toString() + "\n" +
+                "  Now you have " + (tasks.getAllTasks().size() - 1) + " tasks in the list.");
```

Comment:
Based on coding standards, it's better to place the line break after a comma or an operator to improve readability.

Reply:
Okay, I will improve on it. Thank you for your comment!